examples/test_yield.py

```diff
@@ -0,0 +1,13 @@
+import pytest
+
+
+def setup_and_teardown():
+    print('\nsetup')
+    yield 42
+    print('\nteardown')
+
+
+@pytest.mark.param_scope(setup_and_teardown, None)
+@pytest.mark.parametrize('x', ['a', 'b', 'c'])
+def test_yield(x, param_scope):
+    assert param_scope == 42
```

Comment:
A test like this could be used to exercise the extra teardown case

```suggestion
    assert param_scope == 42



def extra_teardown():
    print('\nextra_teardown')



@pytest.mark.param_scope(setup_and_teardown, extra_teardown)
@pytest.mark.parametrize('x', ['a', 'b', 'c'])
def test_yield_extra_teardown(x, param_scope):
    assert param_scope == 42
```

Reply:
I think I'll remove the ability to add an extra teardown.